x-pack/plugins/spaces/public/copy_saved_objects_to_space/components/space_result_details.scss

```diff
@@ -11,3 +11,15 @@
   // Constrains name to the flex item, and allows for truncation when necessary
   min-width: 0;
 }
+
+.spcCopyToSpaceResultDetails__selectControl {
+  margin-left: 30px;
+}
+
+.spcCopyToSpaceResultDetails__selectControl.euiAccordion-isClosed {
+  .euiAccordion__childWrapper {
+    visibility: hidden;
+    opacity: 0;
+    height: 0;
+  }
+}
```

Comment:
Looks like you ran into some trouble with EuiAccordion. Mind expanding on this, perhaps making an issue in the EUI repo?

Reply:
So, I didn't exactly have trouble with the accordion itself.

I wanted nested accordion-like behavior (expanding and contracting with animations), but I didn't want the entirety of this  row to be a clickable element.

<details>
<summary>See GIF</summary>

![image](https://user-images.githubusercontent.com/5295965/90162443-9e43d300-dd62-11ea-84d5-14a93f946c83.gif)
</details>

At the time, I thought that it was best to reuse the existing Accordion CSS classes to better maintain the same behavior (in case Accordion's animation speed changes in the future, for example).

What would you recommend I do here? It didn't seem EUI-issue-worthy to me, as I'm really doing something different than the Accordion was intended for. I could simply create my own separate CSS classes if reusing the Accordion's classes like this is frowned upon.

Reply:
Usually we don't recommend using `.eui` classes directly because we do change them from time to time which would cause breaks for you. 

I would certainly suggest creating your own classes and handling the show/hide styles manually. If you just want to replicate the transition we're using it's:

```scss
  transition:
    height $euiAnimSpeedNormal $euiAnimSlightResistance,
    opacity $euiAnimSpeedNormal $euiAnimSlightResistance;
```

But you bring up an interesting use-case of needing some sort of just generic show/hide (or disclosure component).